Step2 doc updated (#369)
EBIvariation/eva-opentargets: docs/manual-curation/step2-manual-curation.md

The goals of the manual curation:
* All traits which are linked to NT expansion (nucleotide repeat expansion) variants must be curated. Those are marked as "NT expansion" in the “Notes” column.
* All traits with occurrence ≥ **10** must be curated. Additionally, if there are less than **200** such terms, then the top 200 terms must be curated.

> * _Suggested previous mapping_ traits should be checked for any terms that have become obsolete since the last iteration. This can be done by filtering then searching for the string EFO\_OBSOLETE

Reviewer: I think we should have a better way for the curators and reviewers to check this bit. Something like a colour code or tag somewhere that we could easily identify.

Reviewer: After all it's just parsing the end of the previous mapping

Author: Agreed. @tcezard & I had a short discussion in which we think that the process I follow for manual curation should be documented step by step. We have scheduled to work on this task and will add this comment to the created JIRA ticket

* For the rest of the traits, we curate as many as possible.

Good mappings must be eyeballed to ensure they are actually good. Alternative mappings for medium or low quality mappings can be searched for using OLS. If a mapping can't be found in EFO, look for a mapping to a HP, ORDO, or MONDO trait name. Most HP/ORDO/MONDO terms will also be in EFO but some are not. These can be imported to EFO using the Webulous submission service.

## Criteria to manually evaluate mapping quality
* Exact string for string matches are _good_
* Slight modifications are _good_ e.g. IRAK4 DEFICIENCY → Immunodeficiency due to interleukin-1 receptor-associated kinase-4 deficiency

> * Subtype to parent are _good_ e.g ACHROMATOPSIA 3 → Achromatopsia but non-EFO exact string matches are prioritized and then set as _IMPORT_

Reviewer: ```suggestion
* Subtype to parent are _good_ e.g ACHROMATOPSIA 3 → Achromatopsia, but **only if** there is not already a non-EFO exact string match for the subtype. If there is one, it should be prioritized and then term set as _IMPORT_
```

* **SKIP** — trait is going to be skipped in this iteration, due to being too non-specific, or just having a low frequency
* **UNSURE** — temporary status; traits to be discussed with reviewers/the team

### Comment field for curation review

> The "Comment" field can be used to enter arbitrary additional information which will be used by reviewers. Precede any text with initials e.g. "BK - example comment"

Reviewer: ```suggestion
The "Comment" field can be used to enter arbitrary additional information which will be used by reviewers. Precede any text with initials e.g. "BK - example comment". Comments should be ordered chronologically in reverse: most recent ones at the top.
```

> Any comments will become available in the Notes field within the next iteration

Reviewer 2: ```suggestion
Any comments will become available in the Notes field within the next iteration.
Comments from previous iteration that needs to be kept for subsequent ones  should be copy/pasted from the Notes to the Comments cell. 
```
